<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QMainWindow" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLCDNumber" name="LCD">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>8</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="2">
           <widget class="QPushButton" name="btn_plus">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn_4">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btn_5">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btn_7">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btn_2">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btn_3">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="btn_0">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="btn_1">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btn_8">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btn_6">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btn_9">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="btn_mp">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_15">
            <property name="minimumSize">
             <size>
              <width>61</width>
              <height>61</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>61</width>
              <height>61</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_eql">
            <property name="minimumSize">
             <size>
              <width>61</width>
              <height>61</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>61</width>
              <height>61</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
